app/components/TopBarBreadcrumbs.tsx

```diff
@@ -0,0 +1,185 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+import cn from 'classnames'
+import { Link, useParams } from 'react-router-dom'
+
+import { PrevArrow12Icon } from '@oxide/design-system/icons/react'
+
+import { Slash } from '~/ui/lib/Slash'
+import { pb } from '~/util/path-builder'
+
+export const TopBarBreadcrumbs = () => {
+  const [, firstPathItem, secondPathItem, thirdPathItem, , fifthPathItem] =
+    window.location.pathname.split('/')
+
+  // console.log(paths, paths.slice(0, -1).join('/'))
+  const { project } = useParams()
+  // if there's no secondPathItem on the silo section, there's no page to go "back" to
+  // the secondPathItem is top-level within the system section and therefore we check for thirdPathItem
+  const isTopLevel = (firstPathItem === 'system' && !thirdPathItem) || !secondPathItem
+  return (
+    <nav
+      className="flex items-center gap-0.5 overflow-clip pr-4 text-sans-md"
+      aria-label="Breadcrumb navigation"
+    >
+      <PrevArrow12Icon
+        className={cn('mx-1.5 flex-shrink-0 text-quinary', isTopLevel && 'opacity-40')}
```

Comment:
Went for opacity to fade out the prev button when it's no longer applicable instead.